README-translated/README-Hindi.md

```diff
@@ -0,0 +1,127 @@
+
+_द ट्यूरिंग वे_ प्रतिलिपि प्रस्तुत करने योग्य, नैतिक और सहयोगी डेटा विज्ञान के लिए एक व्यावहारिक मार्गदर्शिका है। 
+हम सभी के लिए सुलभ, समझने योग्य और प्रभावी डेटा विज्ञान प्राप्त करने के लिए योगदानकर्ताओं के एक विविध समुदाय का समर्थन करते हैं। 
+हमारा लक्ष्य शिक्षा जगत, उद्योग और सार्वजनिक क्षेत्र के शोधकर्ताओं और डेटा वैज्ञानिकों को अपनी परियोजनाओं की शुरुआत में सभी जानकारी प्रदान करना है ताकि यह सुनिश्चित किया जा सके कि अंत में उन्हें पुन: पेश करना आसान हो।
+
+[![द ट्यूरिंग वे प्रोजेक्ट एक किताब, एक समुदाय, एक ओपन सोर्स (Open Source) प्रोजेक्ट और एक सहयोगी संस्कृति है। यह चार दृष्टांतों में दिखाया गया है, पहला द ट्यूरिंग वे पुस्तक दिखा रहा है, दूसरा समुदाय कैसे विकसित हो सकता है, तीसरा दो लोगों को पुल अनुरोध पर सहयोग कर रहा है, और अंतिम संतुलन दिखा रहा है जहां प्रतिलिपि प्रस्तुत करने की संख्या प्रकाशित की संख्या से अधिक मूल्यवान है दस्तावेज।](../book/website/figures/README_imgs/README_turingway.png)](https://docs.google.com/presentation/d/13Nm8LcRW87ffxEugGEs5j6HKQEvhiuH8b7MfIdX7MpI/edit#slide=id.p1).
+
+*द ट्यूरिंग वे एक किताब, एक समुदाय और एक वैश्विक सहयोग है।.*
+
+छात्रों, शोधकर्ताओं, सॉफ्टवेयर इंजीनियरों, परियोजना नेताओं और संस्थापक टीमों सहित सभी हितधारकों को डेटा विज्ञान पुनरुत्पादन में उनकी भूमिकाओं और जिम्मेदारियों को समझने के लिए द ट्यूरिंग वे का उपयोग करने के लिए आमंत्रित किया जाता है।
+आप [ऑनलाइन](https://github.com/alan-turing-institute/the-turing-way) पुस्तक पढ़ सकते हैं, परियोजना में योगदान देने के लिए हमारे [गाइड](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md) में वर्णित परियोजना में योगदान दे सकते हैं और सभी सामग्रियों का पुन: उपयोग करते हैं ([लाइसेंस देखें](https://github.com/alan-turing-institute/the-turing-way/blob/master/LICENSE.md))।
+
+[![ट्यूरिंग तरीके की ऑनलाइन पुस्तक का स्क्रीनशॉट। यह पुस्तक की शुरुआत में द ट्यूरिंग वे दृष्टांतचित्रों में से एक को भी दिखाता है। इस दृष्टांत में, हम विभिन्न डेटा विज्ञान कौशल के लिए दुकानों के साथ एक सड़क या सड़कें देख सकते हैं। जिन दुकानों में लोग अपनी खरीदारी सूची लेकर प्रवेश कर सकते हैं, उन्हें क्या चाहिए वो लेकर बाहर जा सकते हैं।](../book/website/figures/README_imgs/README_book.png)](https://the-turing-way.netlify.app/welcome.html)
+
+*द ट्यूरिंग वे ऑनलाइन पुस्तक से स्क्रीनशॉट ([इस छवि का उपयोग एक प्रस्तुति में करें](https://drive.google.com/file/d/1wJR664YECSc8b_RSHeyVjDlHs-Ls9lkc/view?usp=sharing))*
+
+द ट्यूरिंग वे 2019 में एक छोटी डेटा विज्ञान मार्गदर्शिका के रूप में शुरू हुआ, और तब से यह तब तक विस्तार हुआ है और आब यह कई विषयोंको (जैसे की: पुनरुत्पादित अनुसंधान, परियोजना डिजाइन, संचार, सहयोग और नैतिक शोध) कवर करने वाले कई गाइडों द्वारा गठित एक पूर्ण मैनुअल बन गया है। प्रत्येक मार्गदर्शिका विभिन्न विषयों पर अध्याय प्रस्तुत करती है जो अच्छे अभ्यासों और अनुशंसाओं को कवर करते हैं। इन अध्यायों को सहयोगियों द्वारा सह-लिखा गया है जो छात्र, शोधकर्ता, शिक्षक, समुदाय के नेता, राजनेता, और विविध पृष्ठभूमि के पेशेवर जिनको अपने पेशे का जीवित अनुभव और समुचा ज्ञान हैं।
```

Comment:
```suggestion
द ट्यूरिंग वे 2019 में एक छोटी डेटा विज्ञान मार्गदर्शिका के रूप में शुरू हुआ, और तब से यह तब तक विस्तार हुआ है और आब यह कई विषयोंको (जैसे की: पुनरुत्पादित अनुसंधान, परियोजना डिजाइन, संचार, सहयोग और नैतिक शोध) कवर करने वाले कई गाइडों द्वारा गठित एक पूर्ण मैनुअल बन गया है। प्रत्येक मार्गदर्शिका विभिन्न विषयों पर अध्याय प्रस्तुत करती है जो अच्छे अभ्यासों और अनुशंसाओं को कवर करते हैं। इन अध्यायों को सहयोगियों द्वारा सह-लिखा गया है जो छात्र, शोधकर्ता, शिक्षक, समुदाय के नेता, राजनेता, और विविध पृष्ठभूमि के पेशेवर जिनको अपने पेशे का जीवित अनुभव और समुचा ज्ञान हैं।
```
 द ट्यूरिंग वे 2019 में एक छोटी डेटा विज्ञान मार्गदर्शिका के रूप में शुरू हुआ, और तब से यह तब तक विस्तार हुआ tha.


-----
तब से iska विस्तार हुआ है और aब यह कई विषयों ka (जैसे की: पुनरुत्पादित अनुसंधान, परियोजना डिजाइन, संचार, सहयोग और नैतिक शोध) varnan करने वाले कई गाइडों द्वारा गठित एक पूर्ण मैनुअल बन गया है। प्रत्येक मार्गदर्शिका विभिन्न विषयों पर अध्याय प्रस्तुत करती है jisme अच्छे अभ्यासों और अनुशंसाओं ka ullekh hai।

- Also in the above sentence- Again, see if we’ve referenced ‘margdarshak’ before or not – or we change this to ‘margdarshakon’ …I would prefer the soln of saying margdarshak (guide) before and simply using ‘guide’ in the rest of the doc.

--

last sentence
इन अध्यायों को सहयोगियों द्वारा सह-लिखा गया है जो छात्र, शोधकर्ता, शिक्षक, समुदाय के नेता, राजनेता, और विविध पृष्ठभूमि के पेशेवर hain, जिनको अपने पेशे का _जीवित_ अनुभव और समुचा ज्ञान हैं।

maybe jivit is not needed in this line.